src/programs/meta.ts

```diff
@@ -0,0 +1,47 @@
+import { Command, CommandHandler, DiscordEvent } from "../event-distribution";
+import { Message, TextChannel } from "discord.js";
+import bot from "../index";
+import Tools from "../common/tools";
+
+const metaEmojiName = "eeeh";
+
+@Command({
+  event: DiscordEvent.MESSAGE,
+  trigger: "!meta",
+  description: "This handler is for the meta command",
+})
+class MetaCommand implements CommandHandler<DiscordEvent.MESSAGE> {
+  async handle(message: Message): Promise<void> {
+    const referencedMessageId = message.reference.messageID;
+    const channelId = message.reference.channelID;
+    const channel = bot.channels.resolve(channelId) as TextChannel;
+    const referenceMessage = await channel.messages.fetch(referencedMessageId);
+
+    if (referenceMessage.author === message.author) {
+      Tools.handleUserError(message, "You cannot meta yourself!");
+      return;
+    }
+
+    const emojiByName = (name: string) =>
+      message.guild.emojis.cache.find((e) => e.name === name);
+    const metaEmoji = emojiByName(metaEmojiName);
+
+    const messageReactions = referenceMessage.reactions.cache.some(
+      (reaction) => reaction.me
+    );
+    if (messageReactions) {
+      Tools.handleUserError(
+        message,
+        "This command was already used on this user!"
```

Comment:
More accurate error:
```suggestion
        "This command was already used on this message!"
```